testData/ontologies/flat.json5

```diff
@@ -318,137 +329,247 @@
             "default": true
         }
     },
+    // Currently adapted from: https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/available-language-packs-for-windows
+    // TODO: This should be platform independent and pinned to some reasonable standard (such as RFC-5646).
     "http://registry.gpii.net/common/language": {
         "schema": {
             "title": "Language",
             "description": "Language of the system",
             "type": "string",
-            "default": "en",
+            "default": "en-US",
             "enum": [
-                "en",
+                "af-ZA",
```

Comment:
Do you think it would be helpful to sort these into alphabetical order based on the actual value rather than the English-language rendition of the language name? It would certainly make validation errors easier to scan and in practice I imagine we will not use the English name when presenting in a UI anyway, but rather the more universal practice of rendering the language's name in its own language.

Reply:
It would make it easier for devs, but IMO the other way makes it easier to display these in @sgithens work.  I can go either way, maybe he sorts them on his end?